x-pack/test/fleet_api_integration/config.ts

```diff
@@ -14,8 +14,7 @@ import { defineDockerServersConfig } from '@kbn/test';
 // This hash comes from the latest successful build of the Snapshot Distribution of the Package Registry, for
 // example: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fpackage-storage/detail/snapshot/74/pipeline/257#step-302-log-1.
 // It should be updated any time there is a new Docker image published for the Snapshot Distribution of the Package Registry.
-export const dockerImage =
-  'docker.elastic.co/package-registry/distribution:b6a53ac9300333a4a45f3f7d350c9aed72061a66';
+export const dockerImage = 'docker.elastic.co/package-registry/distribution:staging';
```

Comment:
The fixed commit was introduced during the time when we had frequent changes. It made sure, any change to the registry will not break CI builds for all of Kibana. By now, things are much more stable. 

I like the idea of frequently updating it. Even though `staging`, more likely `production` should always work, I kind of like the fact that a CI build is always reproducible. The part I don't like is that we have to update this manually too often and catch bugs way too late.

We had similar issues for the stack versions for Beats / Agents builds. Now we have a script in place that on a predefined period updates the version hash with a pull request and it automatically gets merged when it passes. I wonder if this would bring us the best of both worlds?

Reply:
++ to above. @jfsiii As we discussed, can we update the image here to a newer version (but still using a fixed build) and get this PR in? We can then iterate on improving the process to update the image we use more frequently.

Reply:
@ruflin @jen-huang Updated to a newer version in https://github.com/elastic/kibana/pull/101911/commits/32a130cbd7662055d7fa1b8684c8d75a96a696af